integration-tests/src/test/scala/za/co/absa/spline/SQLCommandsSpec.scala

```diff
@@ -58,9 +60,9 @@ class SQLCommandsSpec extends AsyncFlatSpec
                   | WHERE id > 1""".stripMargin))
 
           } yield {
-            plan1.operations.write.outputSource should equalToUri(s"file:$warehouseDir/sourcetable")
+            plan1.operations.write.outputSource should equalToUri(s"${warehouseUri.toString}/sourcetable")
             plan2.operations.reads.get.head.inputSources.head should equalToUri(plan1.operations.write.outputSource)
-            plan2.operations.write.outputSource should equalToUri(s"file:$warehouseDir/targettable")
+            plan2.operations.write.outputSource should equalToUri(s"${warehouseUri.toString}/targettable")
```

Comment:
You don't need `toString` here it's called automatically when using `s"$warehouseUri"`

Reply:
Fixed in commit e54f5ef9a17cf0c9ad82fbc7081d401b31a933b3.